docs/faq.md

```diff
@@ -92,6 +92,64 @@ Add the following custom CSS to your `user.css` or via the settings menu:
 }
 ```
 
+### How to use a custom font on the landing page?
+
+For example, if you want to change it to Roboto, add the following custom CSS to the your `user.css` or via the settings menu:
+
+```css
+@font-face {
+    font-family: "Roboto";
+    font-style: normal;
+    font-weight: 200;
+    src: local(""), url("<font URL for weigth 200>") format("woff2");
+}
+@font-face {
+    font-family: "Roboto";
+    font-style: normal;
+    font-weight: 300;
+    src: local(""), url("<font URL for weigth 300>") format("woff2");
+}
+@font-face {
+    font-family: "Roboto";
+    font-style: normal;
+    font-weight: 400;
+    src: local(""), url("<font URL for weigth 400>") format("woff2");
+}
+@font-face {
+    font-family: "Roboto";
+    font-style: normal;
+    font-weight: 500;
+    src: local(""), url("<font URL for weigth 500>") format("woff2");
+}
+@font-face {
+    font-family: "Roboto";
+    font-style: normal;
+    font-weight: 700;
+    src: local(""), url("<font URL for weigth 700>") format("woff2");
+}
+
+#logo h1 span,
+#logo h1,
+#intro,
+.menu .menu-item {
+    font-family: "Roboto", sans-serif !important;
+}
+
+#intro .subtitle,
+#logo .subtitle {
+    margin-top: 0 !important;
+}
```

Comment:
Is this required or related to using an alternative font?

Reply:
I assumed so. But then I recommend to keep it out of the generic template and present it as part of the template for Google Roboto. Also add a remark about it which explains it.

Reply:
I'm fine with it.